docs/en/sql-reference/functions/string-search-functions.md

```diff
@@ -83,6 +88,36 @@ Result:
 └───────────────────────────────┘
 ```
 
+**Examples**
```

Comment:
We already have this title above, why do we need it here again?

Reply:
Maybe we should put here something like this "Examples for POSITION(needle IN haystack) syntax"?
Or separation examples one from another syntax is not necessary?

Reply:
> We already have this title above, why do we need it here again?

Maybe we should put here something like this "Examples for POSITION(needle IN haystack) syntax"?
Or separation examples one from another syntax is not necessary?

Reply:
> Maybe we should put here something like this "Examples for POSITION(needle IN haystack) syntax"?

Good idea.